src/index.tsx

```diff
@@ -4,10 +4,12 @@ import { HashRouter as Router } from 'react-router-dom';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
-import { App } from './App';
+import { Root } from './Root';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <html className="has-navbar-fixed-top">
+    <Router>
+      <Root />
+    </Router>
+  </html>,
```

Comment:
Error: Rendering the `html` element directly in React is not valid. Instead, remove the `html` element and let the `Router` directly wrap the `Root` component. This will ensure the component tree is correctly structured.